src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lot = new Lottery();
```

Comment:
no need to obfuscate your code :)
```suggestion
        Lottery lottery = new Lottery();
```